src/com/lundekhan/summarizer/SummarizerRoute.kt

```diff
@@ -30,23 +34,90 @@ fun Route.summarizerRoute(): Route = route("/smry") {
         summarizeReq.lines is Int -> summarizeModel.summarize(summarizeReq.text, summarizeReq.lines)
         else -> summarizeModel.summarize(summarizeReq.text, 0.2)
     }
+    // TODO make reduction & model be on one line if possible (FLEX / section)
+    fun SECTION.summarizeForm(selectedItem: String, currentText: String?, reduction: String) {
+        form(method = FormMethod.post) {
+            acceptCharset = "utf-8"
+            header {
+                style = "padding:0"
+                h3 { +"smry." }
+                p { +"Summarize your articles" }
+            }
+            section {
+                label { +"Model:" }
+                label { +"Reduction (X = lines, 0.Y = percentage)" }
+            }
+            section {
+                select {
+                    name = "model"
+                    (listOf(selectedItem) + models.filterNot { it == selectedItem }).forEach { modelName ->
+                        option {
+                            value = modelName
+                            +modelName
+                        }
+                    }
+                }
 
-    post {
-        val summarizeReq = call.receive<SummarizeReq>()
-        val summary = summarize(summarizeReq, summarizer)
-        call.respond(resultResponse(summary))
+                numberInput(name="reduction") {
+                    min = "0"
+                    step = "0.05"
+                    value = reduction
+                }
+            }
+            label { +"Input:" }
+            textArea(rows = "3") {
+                placeholder = "Insert article to summarize"
+                name = "text"
+                if (currentText != null) +currentText
+            }
```

Comment:
Borde ha en annan default storlek

Reply:
Tips?

Reply:
Pass, just den borde ju vara lika bred som kortet(ofc med lite på varje sida). Sen kanske 4-6 rader lång kanske? kanske mer? 
En extra feature på den är att kunna läsa in en fil ?

Reply:
Och hemsida!